locales/en/breaches.ftl

```diff
@@ -109,18 +129,22 @@ breach-checklist-dob-body = Birth dates are easy to find in public records, and
 
 ## Prompts the user for changes when there is a breach detected of phone number
 
-breach-checklist-phone-header = Protect your phone number with a masking service like <a>{-brand-relay}</a>, which hides your true phone number. 
+# Variables:
+#   $firefoxRelayLink (string) - a link to Firefox Relay, with { -breach-checklist-link-firefox-relay } as the label
+breach-checklist-phone-header-2 = Protect your phone number with a masking service like { $firefoxRelayLink }, which hides your true phone number.
 
 ## Prompts the user for changes when there is a breach detected of security questions
 
-# NOTE: { $breachedCompanyUrl } is a placeholder for the URL to the website of the company where the breach occurred 
-breach-checklist-sq-header = Update your security questions on <a>{$breachedCompanyUrl}</a>. 
+# { $breachedCompanyLink } will link to the website of the company where the breach occurred
+breach-checklist-sq-header-2 = Update your security questions on {$breachedCompanyLink}.
 breach-checklist-sq-body = Use long, random answers, and store them somewhere safe. Do this anywhere else you’ve used the same security questions.
 
 ## Prompts the user for changes when there is a breach detected of historical password
 
 breach-checklist-hp-header = Create unique, strong passwords for any account where you’ve re-used passwords.
-breach-checklist-hp-body = A password manager like <a>{-brand-firefox} Password Manager</a> (which is free and built-in to the {-brand-firefox} browser) can help you keep track of all your passwords and access them securely from all your devices. 
+# Variables:
+#   $passwordManagerLink (string) - a link to the password manager documentation, with { -breach-checklist-link-password-manager } as the label
+breach-checklist-hp-body-2 = A password manager like { $passwordManagerLink } (which is free and built-in to the {-brand-firefox} browser) can help you keep track of all your passwords and access them securely from all your devices.
```

Comment:
Yes, because the previous version did not actually insert a link, just the `<a>`. Unfortunately, with the current setup of Monitor, AFAIK we're unable to dynamically update the tags like we can do in Relay (which uses `@fluent/react`), for example.

Reply:
I've already implemented the alternative you suggested here:
https://github.com/mozilla/blurts-server/blob/df622d246bcb2b2212e3de22b016ae1a368cc1ea/src/views/partials/add-email.js#L23-L26
with corresponding translation string here:
https://github.com/mozilla/blurts-server/blob/df622d246bcb2b2212e3de22b016ae1a368cc1ea/locales/en/add-email.ftl#L21

Note the example is more involved due to multiple variables in the string, but if this pattern is preferable to localizers we could try using it instead?

Reply:
Yeah, I also just realised that it would involve a larger refactoring here, since we're currently setting the same variables for every resolution string. Although with multiple links in a single string, we'd probably want to use different tags anyway, e.g. `<experian-link>Experian</experian-link>`. But since we'd just be doing a search&replace rather, I feel like that could be somewhat brittle, so I'd lean towards the current approach.

Reply:
The spacing hasn't been fixed here.